raiden-cli/src/utils/channels.ts

```diff
@@ -26,17 +26,16 @@ export function filterChannels(
   return filteredChannels;
 }
 
-/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Is this related to the new typescript-eslint rules? Or why can you remove that?

Reply:
Yes. Linting actually complains on this line on my machine freshly `npm ci`ed due to this rule not existing anymore. Check https://github.com/raiden-network/light-client/pull/1612#discussion_r433815089 for details